### ISSUE TITLE
PandaDoc::Document.send Has Response Error Issue

### ISSUE THREAD

**Issue by jonmchan** [opti/panda_doc#9]
Error occurs when using the `send` command:

```
PandaDoc::Document.send('QATL5BCBAughzSwaPZKgsT', message: "A message to include into the email")

/usr/local/bundle/gems/dry-types-1.5.1/lib/dry/types/schema.rb:384:in `block in resolve_missing_keys': [PandaDoc::Objects::Recipient.new] :has_completed is missing in Hash input (Dry::Struct::Error)
/usr/local/bundle/gems/dry-types-1.5.1/lib/dry/types/schema.rb:384:in `block in resolve_missing_keys': :has_completed is missing in Hash input (Dry::Types::MissingKeyError)
```

The document is successfully sent, however it appears the library has an issue parsing the response from the call. Pandadoc shows no issue as the document moves to a "sent" state.

**Comment by jonmchan** [comment]
This is the response that I get hitting the endpoint:

```
{
    "id": "PajNoR6fkyWZTvpkM7t5ve",
    "name": "[DEV] My Contract3",
    "status": "document.sent",
    "date_created": "2022-03-07T19:01:28.731001Z",
    "date_modified": "2022-03-07T19:02:48.192273Z",
    "date_completed": null,
    "expiration_date": "2022-05-06T19:02:48.009780Z",
    "version": "2",
    "uuid": "PajNoR6fkyWZTvpkM7t5ve",
    "recipients": [
        {
            "id": 128360233,
            "first_name": "User",
            "last_name": "One",
            "email": "test@example.com",
            "recipient_type": "CC",
            "signing_order": null,
            "shared_link": "https://app.pandadoc.com/document/abc123"
        },
        {
            "id": 128360232,
            "first_name": "User",
            "last_name": "Two",
            "email": "test2@example.com",
            "recipient_type": "signer",
            "signing_order": null,
            "shared_link": "https://app.pandadoc.com/document/abc123"
        }
    ]
}
```

**Comment by opti** [comment]
Fixed in [v0.6.0](https://github.com/opti/panda_doc/blob/main/CHANGELOG.md#060-2022-05-04)
